h/stcbase.h

```diff
@@ -42,6 +42,7 @@
 
 #define STC_MODULE_GENERIC          0x00000000
 #define STC_MODULE_JEDHTTP          0x00010000
+#define STC_MODULE_ZSS              0x00020000
```

Comment:
@ifakhrutdinov does using 0x00020000 this slot for STC_MODULE_ZSS looks good to you?

Reply:
changed the module name to STC_MODULE_BACKGROUND